### PR TITLE
helm updates for new aws ebs csi plugin and new managed  dev cluster

### PR DESCRIPTION
**Pull request by webcoderz**
helm updates for new aws ebs csi plugin and managed cluster with 110 pod limit, and updated name in gha to new managed cluster

### CODE REVIEW

File: charts/graphistry-helm/eks-dev-values-second.yaml [hunk 1]
```diff
@@ -1,18 +1,18 @@
-domain: eks-dev2.grph.xyz 
+domain: g-dev.grph.xyz
```
[the remark this answers is not in the document]
Comment (webcoderz) [reply charts/graphistry-helm/eks-dev-values-second.yaml:1]: oh that was when was doing namespace for daniel that didnt pan out.. i can make an example-aws-values.yaml like i did for azure and microk8s

[the remark this answers is not in the document]
Comment (lmeyerov) [reply charts/graphistry-helm/eks-dev-values-second.yaml:1]: maybe we can put all the ones we don't expect anyone to use into another repo, or at least into a folder they can ignore?

[the remark this answers is not in the document]
Comment (webcoderz) [reply charts/graphistry-helm/eks-dev-values-second.yaml:1]: okay thats a good idea

[the remark this answers is not in the document]
Comment (lmeyerov) [reply charts/graphistry-helm/eks-dev-values-second.yaml:1]: like `deployments/{base,experiments}`

[the remark this answers is not in the document]
Comment (webcoderz) [reply charts/graphistry-helm/eks-dev-values-second.yaml:1]: like ~/internal-graphistry-overrides for us  , ~/example-overrides for users

File: charts/graphistry-helm/grafana-values.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+grafana:
```
Comment (lmeyerov) [inline charts/graphistry-helm/grafana-values.yaml:1]: maybe we carve out these into a plugins folder or something as well?

File: charts/graphistry-helm/values.yaml [hunk 3]
```diff
@@ -76,6 +79,16 @@ multiNode: false
...
+
+# django drf-jwt, jwt token timeout (seconds)
+jwtExpirationDelta : "3600"
+
+# ENABLE django silk - performance analysis library
+enableDjangoSilk : "False"
+
+
 #restart policy
 restartPolicy: Always
```
Comment (lmeyerov) [inline charts/graphistry-helm/values.yaml:93]: I think also an equiv for `COOKIE_SECURE=True` (default off,TLS should generally flip on auto or at least manually)

**Review by lmeyerov — CHANGES_REQUESTED** [review]
main missing thing afaict is opt-in to `COOKIE_SECURE=True` for nexus

too big for this PR, but this looks confusing for folks to know which files to use for what b/c a lot of combining core, extension, telemetry, experimental, internal, and sample variants into megafolders... and no clear pointers+dillineation of which to actually use. imagine someone getting started without having talked w/ any of us...

File: charts/graphistry-helm/grafana-values.yaml (hunk 2 above)
Comment (webcoderz) [reply charts/graphistry-helm/grafana-values.yaml]: put it in ~/overrides/plugins

**Comment by webcoderz** [comment]
Ok will add the cookie secure stuff if tls is set to true in nexus else false by default, this is the structure of a helm template so no clue as to how to minimize complexity, will expand the docs.

File: charts/graphistry-helm/values.yaml (hunk 3 above)
Comment (webcoderz) [reply charts/graphistry-helm/values.yaml:93]: added here: https://github.com/graphistry/graphistry-helm/pull/20/commits/3877b1522a65d3dedc7c44d5b6e1b5fa01b388fb

**Comment by lmeyerov** [comment]
cool, can we merge this?

re:docs, twos thoughts are:
*  "the way to eat an elephant is one bite at a time": we can break up into pieces, and only do one at a time... but well (cc @acald-creator  !)
* ... reflect on your recent deploy for which / when

maybe:
- metaticket listing docs pages that would help
- and explicitly state which to prioritize

then easier to do out for you + others as we go!

**Comment by webcoderz** [comment]
Ya good idea. Should be ready to merge will go ahead and do it

**Pull request merged**